src/web/mjs/connectors/Qiman5.mjs

```diff
@@ -53,6 +53,19 @@ export default class Qiman5 extends SinMH {
         `;
     }
 
+    canHandleURI(uri) {
+        super.initialize()
+            .then(() => {
+                return uri.href.includes(this.url);
+            });
+    }
+
+    async _initializeConnector() {
+        const request = new Request(this.url, this.requestOptions);
```

Comment:
What if the current `this.url` is already wrong, will this still work?

Reply:
this.url is the "portal" url, like for many connectors. The difference is its doesnt use redirect, people must click on a single link on the page. How should i do?